apps/passport-client/src/zapp/ZappServer.ts

```diff
@@ -234,15 +238,69 @@ class PODServer extends BaseZappServer implements ZupassPOD {
       console.log(e);
       throw e;
     }
-    const allPCDs = this.getContext().getState().pcds.getAll();
-    const pods = allPCDs
-      .filter((pcd) => pcd.type === PODPCDTypeName)
-      .map((pcd) => (pcd as PODPCD).pod);
+    const allPCDs = this.getContext()
+      .getState()
+      .pcds.getAllPCDsInFolder(ZAPP_POD_SPECIAL_FOLDER_NAME);
+    const pods = allPCDs.filter(isPODPCD).map((pcd) => pcd.pod);
 
     const result = q.query(pods);
 
     return result.matches.map((match) => match.serialize());
   }
+
+  @safeInput(ZupassAPISchema.shape.pod.shape.insert.parameters())
+  /**
+   * Insert a POD into the PCD collection.
+   */
+  public async insert(serializedPod: string): Promise<void> {
+    const pod = POD.deserialize(serializedPod);
+    const id = uuidv4();
+    const podPCD = new PODPCD(id, pod);
+    const serializedPCD = await PODPCDPackage.serialize(podPCD);
+    this.getContext().dispatch({
+      type: "add-pcds",
+      // This is probably not where we want to store these, but it's not clear
+      // how much the folder concept should be visible to the API.
+      // We could add a "meta" parameter which could contain a "zupassFolder"
+      // record, for instance, but this begins to couple the API too closely to
+      // the specifics of Zupass.
+      // In the meantime, however, PODs stored in this way can be retrieved by
+      // Zapps using the query API.
```

Comment:
Not blocking for now to get a working V1 to test, but for Devcon, each app will want the option to have its own folder. There's been talk about collections vs. folders as the abstraction, but realistically given our constraints for Devcon I think sticking with folders is the safest bet right now.